_data/email.yml

```
@@ -9,13 +9,13 @@ websites:
       - u2f
     doc: https://help.aol.com/articles/2-step-verification-stronger-than-your-password-alone
 
-  - name: FastMail
+  - name: Fastmail
     url: https://www.fastmail.com/
     img: fastmail.png
     tfa:
       - sms
       - totp
-      - hardware
+      - u2f
```

Actually they refer to both u2f and the proprietary hardware system of Yubico called "Yubikey OTP"
```suggestion
      - hardware
      - u2f
```